jsk_baxter_robot/baxtereus/baxter-interface.l

```diff
@@ -246,6 +246,21 @@
 		   (setq tms (append tms (list 50))))
      (setq ctype (or ctype controller-type))  ;; use default if ctype is nil
 		 (send-super :angle-vector-sequence avs tms ctype start-time :scale scale :min-time min-time))
+  
+  (:ros-state-callback
+   (msg)
+   (let ((robot-state-names (cdr (assoc :name robot-state))) (robot-msg-names (send msg :name)) (torso-index))
+     ;;Remove toros_t0 . We think this is not the rotational-joint
+     (setq torso-index (position "torso_t0" robot-msg-names :test #'string=))
+     (send msg :name ( remove "torso_t0" robot-state-names :test #'string=))
```

Comment:
(when torso-index
  (send msg :name (remove ...))
  (dolist (key '(:....
)

may ok, position returns nil when it is not found and when 0 is true